### PR TITLE
Rename binary-com to deriv-com

### PR DESCRIPTION
**Pull request by mariovw-deriv**
## Description

There are a few references to the deprecated repo binary-com. This has been replaced by deriv-com

## Cards

https://app.clickup.com/t/20696747/DAPI-805

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -10,12 +10,12 @@
...
     test_suite='tests',
-    url='https://github.com/binary-com/python-deriv-api',
+    url='https://github.com/deriv-com/python-deriv-api',
     project_urls={
-        'Bug Tracker': "https://github.com/binary-com/python-deriv-api/issues",
-        'Documentation': "https://binary-com.github.io/python-deriv-api",
-        'Source Code': "https://github.com/binary-com/python-deriv-api",
-        'Changelog': "https://github.com/binary-com/python-deriv-api/blob/master/CHANGELOG.md"
+        'Bug Tracker': "https://github.com/deriv-com/python-deriv-api/issues",
+        'Documentation': "https://deriv-com.github.io/python-deriv-api",
```
Comment (raju-deriv) [inline setup.py:16]: @mariovw-deriv is it right? we have used two different endpoint. https://deriv-com.github.io (not accessible) or https://github.com/deriv-com/

Comment (mariovw-deriv) [reply setup.py:16]: But this https://deriv-com.github.io/python-deriv-api/ is accessible. Is it a redirect?

Comment (raju-deriv) [reply setup.py:16]: Yes, trying to redirect to deriv.com. 
Screenshot 2024-10-30 at 4 33 42 PM

File: scripts/regen-py.pl [hunk 2]
```diff
@@ -114,7 +114,7 @@ sub needs_method_arg {
     return 1;
 }
 
-emit_functions($ENV{BINARYCOM_API_SCHEMA_PATH} // '/home/git/binary-com/deriv-developers-portal/config/v3');
+emit_functions($ENV{BINARYCOM_API_SCHEMA_PATH} // '/home/git/deriv-com/deriv-developers-portal/config/v3');
```
Comment (mukesh-deriv) [inline scripts/regen-py.pl]: We dont have this repo in deriv-com and the one in binary-com is already archived. should this still exists?

Comment (mariovw-deriv) [reply scripts/regen-py.pl]: it should be deriv-api-docs

**Pull request merged**